### PR TITLE
Add gst-plugins-good patch for upstream issue #528

### PR DESCRIPTION
**Pull request by xjuan**
Add gtkgstwidget-add-ready-to-show-signal.patch
https://gitlab.freedesktop.org/gstreamer/gst-plugins-good/issues/528

https://phabricator.endlessm.com/T24812

### CODE REVIEW

File: generate-manifest.py [hunk 1]
```diff
@@ -103,6 +103,16 @@ def edit_manifest(data, arch, branch, runtime_version):
         ],
     }
 
+    gst_plugins_good_patches = {
+        'all': [
+            'gtkgstwidget-add-ready-to-show-signal.patch',
+        ],
+        'arm': [
```
Comment (cfoch) [inline generate-manifest.py:110]: This is not used. I think this can be removed.

File: generate-manifest.py [hunk 2]
```diff
@@ -103,6 +103,16 @@ def edit_manifest(data, arch, branch, runtime_version):
...
+        ],
+        'x86_64': [
```
Comment (cfoch) [inline generate-manifest.py:112]: Same here.

File: generate-manifest.py [hunk 3]
```diff
@@ -116,7 +126,12 @@ def edit_manifest(data, arch, branch, runtime_version):
                 gtk_module['sources'].append({ 'type': 'patch', 'path': patch })
             data['modules'].insert(0, gtk_module)
             break
-
+        if m['name'] == 'gstreamer-plugins-good':
+            gst_module = m
+            for patch in (gst_plugins_good_patches[arch] + gst_plugins_good_patches['all']):
```
Comment (cfoch) [inline generate-manifest.py:131]: `for patch in gst_plugins_good_patches['all']:`

**Comment by cosimoc** [comment]
I'm merging this as-is since it's copy-pasted from other similar sections in the file.

**Pull request merged**